wallet/service/v2/connections/generate_api_token.go

```diff
@@ -78,6 +78,10 @@ func (h *GenerateAPITokenHandler) Handle(ctx context.Context, params GenerateAPI
 	return tokenDescription.Token, nil
 }
 
+func truncateToSeconds(t time.Time) time.Time {
+	return time.Date(t.Year(), t.Month(), t.Day(), t.Hour(), t.Minute(), t.Second(), 0, t.Location())
+}
```

Comment:
Why not use: `t.Truncate(time.Second)` ?

Reply:
Because it's not advised by the standard documentation:

https://pkg.go.dev/time#example-Time.Truncate

```golang
// To round to the last midnight in the local timezone, create a new Date.
midnight := time.Date(t.Year(), t.Month(), t.Day(), 0, 0, 0, 0, time.Local)
_ = midnight
```

Reply:
_* if we are working with the local timezones_